resources/sonar.properties.tmpl

```diff
@@ -0,0 +1,30 @@
+sonar.log.level={{ default .Env.SONAR_LOG_LEVEL "INFO" }}
+
+sonar.web.javaOpts=-Xmx{{ default .Env.SONAR_WEB_MAXIMUM_MEMORY "1g" }} \
+  -Xms{{ default .Env.SONAR_WEB_MINIMUM_MEMORY "256m" }} \
+  -XX:+HeapDumpOnOutOfMemoryError \
+  -Djava.net.preferIPv4Stack=true
+
+sonar.ce.javaOpts=-Xmx{{ default .Env.SONAR_CE_MAXIMUM_MEMORY "1g" }} \
+  -Xms{{ default .Env.SONAR_CE_MINIMUM_MEMORY "256m" }} \
+  -XX:+HeapDumpOnOutOfMemoryError \
+  -Djava.net.preferIPv4Stack=true
+
+sonar.search.javaOpts=-Xmx{{ default .Env.SONAR_ES_MAXIMUM_MEMORY "2g" }} \
+  -Xms{{ default .Env.SONAR_SE_MINIMUM_MEMORY "512m" }} \
+  -Xss{{ default .Env.SONAR_SE_LIMIT_MEMORY "1m" }} \
+  -Djava.net.preferIPv4Stack=true \
+  -XX:+UseParNewGC \
+  -XX:+UseConcMarkSweepGC \
+  -XX:CMSInitiatingOccupancyFraction=75 \
+  -XX:+UseCMSInitiatingOccupancyOnly \
+  -XX:+HeapDumpOnOutOfMemoryError
+
+{{ if eq .Env.SSO_ENABLED "True" }}
```

Comment:
This bit fails when trying to start up the container as we haven't defaulted the value in the Dockerfile:
```2018/07/26 10:02:00 template error: template: sonar.properties.tmpl:23:6: executing "sonar.properties.tmpl" at <eq .Env.SSO_ENABLED ...>: error calling eq: invalid type for comparison
2018/07/26 10:02:01 template error: template: sonar.properties.tmpl:23:6: executing "sonar.properties.tmpl" at <eq .Env.SSO_ENABLED ...>: error calling eq: invalid type for comparison
2018/07/26 10:02:01 template error: template: sonar.properties.tmpl:23:6: executing "sonar.properties.tmpl" at <eq .Env.SSO_ENABLED ...>: error calling eq: invalid type for comparison
2018/07/26 10:02:02 template error: template: sonar.properties.tmpl:23:6: executing "sonar.properties.tmpl" at <eq .Env.SSO_ENABLED ...>: error calling eq: invalid type for comparison
2018/07/26 10:02:04 template error: template: sonar.properties.tmpl:23:6: executing "sonar.properties.tmpl" at <eq .Env.SSO_ENABLED ...>: error calling eq: invalid type for comparison
2018/07/26 10:02:06 template error: template: sonar.properties.tmpl:23:6: executing "sonar.properties.tmpl" at <eq .Env.SSO_ENABLED ...>: error calling eq: invalid type for comparison
```